app/util/data_preparation/confluence/prepare-data.py

```diff
@@ -90,6 +90,13 @@ def write_test_data_to_files(dataset):
     __write_to_file(CONFLUENCE_USERS, users)
 
 
+def __check_collaborative_editing_enable(confluence_api):
+    status = confluence_api.get_collaborative_editing_status()
+    if not all(status.values()):
+        raise Exception('Synchrony plugin and/or the collaborative editing feature are disabled in Confluence. '
+                        'Please turn collaborative editing back on.')
```

Comment:
Better wording here would be:
`Please turn on collaborative editing in Confluence System Preferences page in order to run DC Apps Performance Toolkit.`

Reply:
Done